src/menu/host_interface.c

```diff
@@ -114,7 +115,29 @@ void main_menu_host_interface(engine_ctx_t *ctx,
       break;
     }
     case 2: {
-      btc_app_main(usb_evt);
+      // TODO: instantiate & configure Bitcoin chain
+      btc_fam_main(usb_evt, get_btc_app());
+      break;
+    }
+    case 3: {
```

Comment:
Do we have buy-in from desktop team to support bitcoin forks using different `app id`?

Reply:
yes